aiplatform/api/AIPlatform.Samples/PredictChatPromptSample.cs

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START aiplatform_sdk_chat]
+
+using Google.Cloud.AIPlatform.V1;
+using wkt = Google.Protobuf.WellKnownTypes;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+public class PredictChatPromptSample
+{
+    public string PredictChatPrompt(
+        string projectId = "your-project-id",
+        string locationId = "us-central1",
+        string publisher = "google",
+        string model = "chat-bison@001"
```

Comment:
Suggestion: Just thinking as an end user, I will very much appreciate if I know all the different models that I can use, so a link  to documentation pointing to different models available will be helpful.

Reply:
I mentioned already in other PR but these samples will be embedded within the docs pages, so given that, I'm not sure if we want to explain more or link to docs.

Reply:
Right Mete. I had prefixed it as a suggestion. If the sample is tied to the docs already, it is fine. Thanks